### DISCUSSION TITLE
Action Center / Notifications - Left side

### DISCUSSION THREAD

**Discussion by ktjbrowne** [valinet/ExplorerPatcher#117] [Ideas]
hi, excellent work. love it!
I use my taskbar on the left, wondering is it possible to move action centre and notifications to display/pop from the bottom left also (and anything that defaultly shows on the right when taskbar is on the bottom)
thanks again,
k.

**Comment by valinet** [comment]
I will look into it hopefully, I am not very familiar with how the "metro"/UWP parts of the shell work. If I figure something out, surely I will add it to the software. Likewise, if you figure something out, do not hesitate to ping me back. Thanks.

**Reply by ktjbrowne** [reply]
understood and thanks.
to clarify my question, I ment the Notifications popup.
thank you.

**Reply by valinet** [reply]
Yeah, I understand, either way, if I figure something out, as I said, for sure it will be included in a subsequent release. Thank you too.
